examples/viewer/viewer.cc

```diff
@@ -151,7 +152,8 @@ static std::string GetBaseDir(const std::string& filepath) {
 
 static bool FileExists(const std::string& abs_filename) {
   bool ret;
-  FILE* fp = fopen(abs_filename.c_str(), "rb");
+  FILE* fp;
+  fopen_s(&fp, abs_filename.c_str(), "rb");
```

Comment:
My visual studio 2017 compiler said fopen is obsolete and prompted me to use fopen_s.